sql/moz-fx-data-shared-prod/app_store_syndicate/dataset_metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: app_store_syndicate
+description: |
+  Syndicated App Store data from Fivetran. User-facing views into this data can be found
+  in the ads dataset
+dataset_base_acl: syndicate
+user_facing: false
+syndication:
+  prod:
+    syndicated_project: "moz-fx-data-bq-fivetran"
+    syndicated_dataset: "app_store"
+  administer_views: false
+workgroup_access: []
```

Comment:
This is probably just copypasta, but from a reading of what this PR is replacing, viewer access here should be set to `workgroup:mozilla-confidential`. As currently configured you're going to define some authorized views into the fivetran dataset within shared-prod that are accessible to `[]` which isn't directly useful without additional authorizations. So if you're deliberately not authorizing the syndicate dataset you'll need to authorize each of the the specific views into this dataset which is supported but has higher propagation delay than just making sure the view and syndicate datasets have the same `workgroup_access`.